### ISSUE TITLE
Registry exiting due to OutOfMemoryError

### ISSUE THREAD

**Issue by jellebens** [Apicurio/apicurio-registry-operator#113]
I installed the registry using the operator 
find the logs below:

```
$ kubectl logs  databus-registry-deployment-l6zkr-7d444c45bb-pcjht -n databus
exec java -Dquarkus.http.host=0.0.0.0 -Djava.util.logging.manager=org.jboss.logmanager.LogManager -javaagent:/opt/agent-bond/agent-bond.jar=jmx_exporter{{9779:/opt/agent-bond/jmx_exporter_config.yml}} -Xmx650m -XX:ParallelGCThreads=1 -XX:ConcGCThreads=1 -Djava.util.concurrent.ForkJoinPool.common.parallelism=1 -XX:CICompilerCount=2 -XX:+UseParallelGC -XX:GCTimeRatio=4 -XX:AdaptiveSizePolicyWeight=90 -XX:MinHeapFreeRatio=20 -XX:MaxHeapFreeRatio=40 -XX:+ExitOnOutOfMemoryError -cp . -jar /deployments/apicurio-registry-storage-streams-1.3.2.Final-runner.jar
__  ____  __  _____   ___  __ ____  ______
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/
2021-02-05 09:39:16,096 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.datasource.username" was provided; it will be ignored; verify that the dependency extension for this configuration is set or you did not make a typo
2021-02-05 09:39:16,099 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.datasource.db-kind" was provided; it will be ignored; verify that the dependency extension for this configuration is set or you did not make a typo
2021-02-05 09:39:16,099 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.datasource.password" was provided; it will be ignored; verify that the dependency extension for this configuration is set or you did not make a typo
2021-02-05 09:39:16,100 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.datasource.jdbc.url" was provided; it will be ignored; verify that the dependency extension for this configuration is set or you did not make a typo
2021-02-05 09:39:16,100 WARN  [io.qua.config] (main) Unrecognized configuration key "quarkus.hibernate-orm.database.generation" was provided; it will be ignored; verify that the dependency extension for this configuration is set or you did not make a typo
2021-02-05 09:39:19,202 INFO  [org.apa.kaf.con.jso.JsonConverterConfig] (main) JsonConverterConfig values:
        converter.type = key
        decimal.format = BASE64
        schemas.cache.size = 0
        schemas.enable = true

2021-02-05 09:39:25,792 INFO  [org.apa.kaf.str.StreamsConfig] (main) StreamsConfig values:
        application.id = databus-registry
        application.server = 10.1.243.45:9000
        bootstrap.servers = [databus-cluster-kafka-bootstrap:9093]
        buffered.records.per.partition = 1000
        built.in.metrics.version = latest
        cache.max.bytes.buffering = 10485760
        client.id =
        commit.interval.ms = 100
        connections.max.idle.ms = 540000
        default.deserialization.exception.handler = class org.apache.kafka.streams.errors.LogAndFailExceptionHandler
        default.key.serde = class org.apache.kafka.common.serialization.Serdes$ByteArraySerde
        default.production.exception.handler = class org.apache.kafka.streams.errors.DefaultProductionExceptionHandler
        default.timestamp.extractor = class org.apache.kafka.streams.processor.FailOnInvalidTimestamp
        default.value.serde = class org.apache.kafka.common.serialization.Serdes$ByteArraySerde
        max.task.idle.ms = 0
        metadata.max.age.ms = 300000
        metric.reporters = []
        metrics.num.samples = 2
        metrics.recording.level = INFO
        metrics.sample.window.ms = 30000
        num.standby.replicas = 1
        num.stream.threads = 2
        partition.grouper = class org.apache.kafka.streams.processor.DefaultPartitionGrouper
        poll.ms = 100
        processing.guarantee = exactly_once
        receive.buffer.bytes = 32768
        reconnect.backoff.max.ms = 1000
        reconnect.backoff.ms = 50
        replication.factor = 1
        request.timeout.ms = 40000
        retries = 0
        retry.backoff.ms = 100
        rocksdb.config.setter = null
        security.protocol = PLAINTEXT
        send.buffer.bytes = 131072
        state.cleanup.delay.ms = 600000
        state.dir = /tmp/kafka-streams
        topology.optimization = none
        upgrade.from = null
        windowstore.changelog.additional.retention.ms = 86400000

2021-02-05 09:39:26,051 INFO  [org.apa.kaf.str.KafkaStreams] (main) stream-client [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c] Kafka Streams version: 2.5.0
2021-02-05 09:39:26,052 INFO  [org.apa.kaf.str.KafkaStreams] (main) stream-client [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c] Kafka Streams commit ID: 66563e712b0b9f84
2021-02-05 09:39:26,161 INFO  [org.apa.kaf.cli.adm.AdminClientConfig] (main) AdminClientConfig values:
        bootstrap.servers = [databus-cluster-kafka-bootstrap:9093]
        client.dns.lookup = default
        client.id = databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-admin
        connections.max.idle.ms = 300000
        default.api.timeout.ms = 60000
        metadata.max.age.ms = 300000
        metric.reporters = []
        metrics.num.samples = 2
        metrics.recording.level = INFO
        metrics.sample.window.ms = 30000
        receive.buffer.bytes = 65536
        reconnect.backoff.max.ms = 1000
        reconnect.backoff.ms = 50
        request.timeout.ms = 30000
        retries = 2147483647
        retry.backoff.ms = 100
        sasl.client.callback.handler.class = null
        sasl.jaas.config = null
        sasl.kerberos.kinit.cmd = /usr/bin/kinit
        sasl.kerberos.min.time.before.relogin = 60000
        sasl.kerberos.service.name = null
        sasl.kerberos.ticket.renew.jitter = 0.05
        sasl.kerberos.ticket.renew.window.factor = 0.8
        sasl.login.callback.handler.class = null
        sasl.login.class = null
        sasl.login.refresh.buffer.seconds = 300
        sasl.login.refresh.min.period.seconds = 60
        sasl.login.refresh.window.factor = 0.8
        sasl.login.refresh.window.jitter = 0.05
        sasl.mechanism = GSSAPI
        security.protocol = PLAINTEXT
        security.providers = null
        send.buffer.bytes = 131072
        ssl.cipher.suites = null
        ssl.enabled.protocols = [TLSv1.2]
        ssl.endpoint.identification.algorithm = https
        ssl.key.password = null
        ssl.keymanager.algorithm = SunX509
        ssl.keystore.location = null
        ssl.keystore.password = null
        ssl.keystore.type = JKS
        ssl.protocol = TLSv1.2
        ssl.provider = null
        ssl.secure.random.implementation = null
        ssl.trustmanager.algorithm = PKIX
        ssl.truststore.location = null
        ssl.truststore.password = null
        ssl.truststore.type = JKS

2021-02-05 09:39:26,441 WARN  [org.apa.kaf.cli.adm.AdminClientConfig] (main) The configuration 'storage.topic' was supplied but isn't a known config.
2021-02-05 09:39:26,441 WARN  [org.apa.kaf.cli.adm.AdminClientConfig] (main) The configuration 'global.id.topic' was supplied but isn't a known config.
2021-02-05 09:39:26,446 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka version: 2.5.0
2021-02-05 09:39:26,447 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka commitId: 66563e712b0b9f84
2021-02-05 09:39:26,447 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka startTimeMs: 1612517966441
2021-02-05 09:39:26,457 INFO  [org.apa.kaf.str.pro.int.StreamThread] (main) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1] Creating restore consumer client
2021-02-05 09:39:26,479 INFO  [org.apa.kaf.cli.con.ConsumerConfig] (main) ConsumerConfig values:
        allow.auto.create.topics = true
        auto.commit.interval.ms = 5000
        auto.offset.reset = none
        bootstrap.servers = [databus-cluster-kafka-bootstrap:9093]
        check.crcs = true
        client.dns.lookup = default
        client.id = databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1-restore-consumer
        client.rack =
        connections.max.idle.ms = 540000
        default.api.timeout.ms = 60000
        enable.auto.commit = false
        exclude.internal.topics = true
        fetch.max.bytes = 52428800
        fetch.max.wait.ms = 500
        fetch.min.bytes = 1
        group.id = null
        group.instance.id = null
        heartbeat.interval.ms = 3000
        interceptor.classes = []
        internal.leave.group.on.close = false
        isolation.level = read_committed
        key.deserializer = class org.apache.kafka.common.serialization.ByteArrayDeserializer
        max.partition.fetch.bytes = 1048576
        max.poll.interval.ms = 300000
        max.poll.records = 1000
        metadata.max.age.ms = 300000
        metric.reporters = []
        metrics.num.samples = 2
        metrics.recording.level = INFO
        metrics.sample.window.ms = 30000
        partition.assignment.strategy = [class org.apache.kafka.clients.consumer.RangeAssignor]
        receive.buffer.bytes = 65536
        reconnect.backoff.max.ms = 1000
        reconnect.backoff.ms = 50
        request.timeout.ms = 30000
        retry.backoff.ms = 100
        sasl.client.callback.handler.class = null
        sasl.jaas.config = null
        sasl.kerberos.kinit.cmd = /usr/bin/kinit
        sasl.kerberos.min.time.before.relogin = 60000
        sasl.kerberos.service.name = null
        sasl.kerberos.ticket.renew.jitter = 0.05
        sasl.kerberos.ticket.renew.window.factor = 0.8
        sasl.login.callback.handler.class = null
        sasl.login.class = null
        sasl.login.refresh.buffer.seconds = 300
        sasl.login.refresh.min.period.seconds = 60
        sasl.login.refresh.window.factor = 0.8
        sasl.login.refresh.window.jitter = 0.05
        sasl.mechanism = GSSAPI
        security.protocol = PLAINTEXT
        security.providers = null
        send.buffer.bytes = 131072
        session.timeout.ms = 10000
        ssl.cipher.suites = null
        ssl.enabled.protocols = [TLSv1.2]
        ssl.endpoint.identification.algorithm = https
        ssl.key.password = null
        ssl.keymanager.algorithm = SunX509
        ssl.keystore.location = null
        ssl.keystore.password = null
        ssl.keystore.type = JKS
        ssl.protocol = TLSv1.2
        ssl.provider = null
        ssl.secure.random.implementation = null
        ssl.trustmanager.algorithm = PKIX
        ssl.truststore.location = null
        ssl.truststore.password = null
        ssl.truststore.type = JKS
        value.deserializer = class org.apache.kafka.common.serialization.ByteArrayDeserializer

2021-02-05 09:39:26,676 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'storage.topic' was supplied but isn't a known config.
2021-02-05 09:39:26,676 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'global.id.topic' was supplied but isn't a known config.
2021-02-05 09:39:26,676 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka version: 2.5.0
2021-02-05 09:39:26,677 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka commitId: 66563e712b0b9f84
2021-02-05 09:39:26,677 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka startTimeMs: 1612517966676
2021-02-05 09:39:26,747 INFO  [org.apa.kaf.str.pro.int.StreamThread] (main) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1] Creating consumer client
2021-02-05 09:39:26,756 INFO  [org.apa.kaf.cli.con.ConsumerConfig] (main) ConsumerConfig values:
        allow.auto.create.topics = false
        auto.commit.interval.ms = 5000
        auto.offset.reset = earliest
        bootstrap.servers = [databus-cluster-kafka-bootstrap:9093]
        check.crcs = true
        client.dns.lookup = default
        client.id = databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1-consumer
        client.rack =
        connections.max.idle.ms = 540000
        default.api.timeout.ms = 60000
        enable.auto.commit = false
        exclude.internal.topics = true
        fetch.max.bytes = 52428800
        fetch.max.wait.ms = 500
        fetch.min.bytes = 1
        group.id = databus-registry
        group.instance.id = null
        heartbeat.interval.ms = 3000
        interceptor.classes = []
        internal.leave.group.on.close = false
        isolation.level = read_committed
        key.deserializer = class org.apache.kafka.common.serialization.ByteArrayDeserializer
        max.partition.fetch.bytes = 1048576
        max.poll.interval.ms = 300000
        max.poll.records = 1000
        metadata.max.age.ms = 300000
        metric.reporters = []
        metrics.num.samples = 2
        metrics.recording.level = INFO
        metrics.sample.window.ms = 30000
        partition.assignment.strategy = [org.apache.kafka.streams.processor.internals.StreamsPartitionAssignor]
        receive.buffer.bytes = 65536
        reconnect.backoff.max.ms = 1000
        reconnect.backoff.ms = 50
        request.timeout.ms = 30000
        retry.backoff.ms = 100
        sasl.client.callback.handler.class = null
        sasl.jaas.config = null
        sasl.kerberos.kinit.cmd = /usr/bin/kinit
        sasl.kerberos.min.time.before.relogin = 60000
        sasl.kerberos.service.name = null
        sasl.kerberos.ticket.renew.jitter = 0.05
        sasl.kerberos.ticket.renew.window.factor = 0.8
        sasl.login.callback.handler.class = null
        sasl.login.class = null
        sasl.login.refresh.buffer.seconds = 300
        sasl.login.refresh.min.period.seconds = 60
        sasl.login.refresh.window.factor = 0.8
        sasl.login.refresh.window.jitter = 0.05
        sasl.mechanism = GSSAPI
        security.protocol = PLAINTEXT
        security.providers = null
        send.buffer.bytes = 131072
        session.timeout.ms = 10000
        ssl.cipher.suites = null
        ssl.enabled.protocols = [TLSv1.2]
        ssl.endpoint.identification.algorithm = https
        ssl.key.password = null
        ssl.keymanager.algorithm = SunX509
        ssl.keystore.location = null
        ssl.keystore.password = null
        ssl.keystore.type = JKS
        ssl.protocol = TLSv1.2
        ssl.provider = null
        ssl.secure.random.implementation = null
        ssl.trustmanager.algorithm = PKIX
        ssl.truststore.location = null
        ssl.truststore.password = null
        ssl.truststore.type = JKS
        value.deserializer = class org.apache.kafka.common.serialization.ByteArrayDeserializer

2021-02-05 09:39:27,193 INFO  [org.apa.kaf.str.pro.int.ass.AssignorConfiguration] (main) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1-consumer] Cooperative rebalancing enabled now
2021-02-05 09:39:27,374 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'storage.topic' was supplied but isn't a known config.
2021-02-05 09:39:27,375 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'admin.retry.backoff.ms' was supplied but isn't a known config.
2021-02-05 09:39:27,416 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'admin.retries' was supplied but isn't a known config.
2021-02-05 09:39:27,416 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'global.id.topic' was supplied but isn't a known config.
2021-02-05 09:39:27,417 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka version: 2.5.0
2021-02-05 09:39:27,417 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka commitId: 66563e712b0b9f84
2021-02-05 09:39:27,417 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka startTimeMs: 1612517967417
2021-02-05 09:39:27,467 INFO  [org.apa.kaf.str.pro.int.StreamThread] (main) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2] Creating restore consumer client
2021-02-05 09:39:27,468 INFO  [org.apa.kaf.cli.con.ConsumerConfig] (main) ConsumerConfig values:
        allow.auto.create.topics = true
        auto.commit.interval.ms = 5000
        auto.offset.reset = none
        bootstrap.servers = [databus-cluster-kafka-bootstrap:9093]
        check.crcs = true
        client.dns.lookup = default
        client.id = databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2-restore-consumer
        client.rack =
        connections.max.idle.ms = 540000
        default.api.timeout.ms = 60000
        enable.auto.commit = false
        exclude.internal.topics = true
        fetch.max.bytes = 52428800
        fetch.max.wait.ms = 500
        fetch.min.bytes = 1
        group.id = null
        group.instance.id = null
        heartbeat.interval.ms = 3000
        interceptor.classes = []
        internal.leave.group.on.close = false
        isolation.level = read_committed
        key.deserializer = class org.apache.kafka.common.serialization.ByteArrayDeserializer
        max.partition.fetch.bytes = 1048576
        max.poll.interval.ms = 300000
        max.poll.records = 1000
        metadata.max.age.ms = 300000
        metric.reporters = []
        metrics.num.samples = 2
        metrics.recording.level = INFO
        metrics.sample.window.ms = 30000
        partition.assignment.strategy = [class org.apache.kafka.clients.consumer.RangeAssignor]
        receive.buffer.bytes = 65536
        reconnect.backoff.max.ms = 1000
        reconnect.backoff.ms = 50
        request.timeout.ms = 30000
        retry.backoff.ms = 100
        sasl.client.callback.handler.class = null
        sasl.jaas.config = null
        sasl.kerberos.kinit.cmd = /usr/bin/kinit
        sasl.kerberos.min.time.before.relogin = 60000
        sasl.kerberos.service.name = null
        sasl.kerberos.ticket.renew.jitter = 0.05
        sasl.kerberos.ticket.renew.window.factor = 0.8
        sasl.login.callback.handler.class = null
        sasl.login.class = null
        sasl.login.refresh.buffer.seconds = 300
        sasl.login.refresh.min.period.seconds = 60
        sasl.login.refresh.window.factor = 0.8
        sasl.login.refresh.window.jitter = 0.05
        sasl.mechanism = GSSAPI
        security.protocol = PLAINTEXT
        security.providers = null
        send.buffer.bytes = 131072
        session.timeout.ms = 10000
        ssl.cipher.suites = null
        ssl.enabled.protocols = [TLSv1.2]
        ssl.endpoint.identification.algorithm = https
        ssl.key.password = null
        ssl.keymanager.algorithm = SunX509
        ssl.keystore.location = null
        ssl.keystore.password = null
        ssl.keystore.type = JKS
        ssl.protocol = TLSv1.2
        ssl.provider = null
        ssl.secure.random.implementation = null
        ssl.trustmanager.algorithm = PKIX
        ssl.truststore.location = null
        ssl.truststore.password = null
        ssl.truststore.type = JKS
        value.deserializer = class org.apache.kafka.common.serialization.ByteArrayDeserializer

2021-02-05 09:39:27,550 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'storage.topic' was supplied but isn't a known config.
2021-02-05 09:39:27,550 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'global.id.topic' was supplied but isn't a known config.
2021-02-05 09:39:27,550 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka version: 2.5.0
2021-02-05 09:39:27,551 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka commitId: 66563e712b0b9f84
2021-02-05 09:39:27,559 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka startTimeMs: 1612517967550
2021-02-05 09:39:27,561 INFO  [org.apa.kaf.str.pro.int.StreamThread] (main) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2] Creating consumer client
2021-02-05 09:39:27,562 INFO  [org.apa.kaf.cli.con.ConsumerConfig] (main) ConsumerConfig values:
        allow.auto.create.topics = false
        auto.commit.interval.ms = 5000
        auto.offset.reset = earliest
        bootstrap.servers = [databus-cluster-kafka-bootstrap:9093]
        check.crcs = true
        client.dns.lookup = default
        client.id = databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2-consumer
        client.rack =
        connections.max.idle.ms = 540000
        default.api.timeout.ms = 60000
        enable.auto.commit = false
        exclude.internal.topics = true
        fetch.max.bytes = 52428800
        fetch.max.wait.ms = 500
        fetch.min.bytes = 1
        group.id = databus-registry
        group.instance.id = null
        heartbeat.interval.ms = 3000
        interceptor.classes = []
        internal.leave.group.on.close = false
        isolation.level = read_committed
        key.deserializer = class org.apache.kafka.common.serialization.ByteArrayDeserializer
        max.partition.fetch.bytes = 1048576
        max.poll.interval.ms = 300000
        max.poll.records = 1000
        metadata.max.age.ms = 300000
        metric.reporters = []
        metrics.num.samples = 2
        metrics.recording.level = INFO
        metrics.sample.window.ms = 30000
        partition.assignment.strategy = [org.apache.kafka.streams.processor.internals.StreamsPartitionAssignor]
        receive.buffer.bytes = 65536
        reconnect.backoff.max.ms = 1000
        reconnect.backoff.ms = 50
        request.timeout.ms = 30000
        retry.backoff.ms = 100
        sasl.client.callback.handler.class = null
        sasl.jaas.config = null
        sasl.kerberos.kinit.cmd = /usr/bin/kinit
        sasl.kerberos.min.time.before.relogin = 60000
        sasl.kerberos.service.name = null
        sasl.kerberos.ticket.renew.jitter = 0.05
        sasl.kerberos.ticket.renew.window.factor = 0.8
        sasl.login.callback.handler.class = null
        sasl.login.class = null
        sasl.login.refresh.buffer.seconds = 300
        sasl.login.refresh.min.period.seconds = 60
        sasl.login.refresh.window.factor = 0.8
        sasl.login.refresh.window.jitter = 0.05
        sasl.mechanism = GSSAPI
        security.protocol = PLAINTEXT
        security.providers = null
        send.buffer.bytes = 131072
        session.timeout.ms = 10000
        ssl.cipher.suites = null
        ssl.enabled.protocols = [TLSv1.2]
        ssl.endpoint.identification.algorithm = https
        ssl.key.password = null
        ssl.keymanager.algorithm = SunX509
        ssl.keystore.location = null
        ssl.keystore.password = null
        ssl.keystore.type = JKS
        ssl.protocol = TLSv1.2
        ssl.provider = null
        ssl.secure.random.implementation = null
        ssl.trustmanager.algorithm = PKIX
        ssl.truststore.location = null
        ssl.truststore.password = null
        ssl.truststore.type = JKS
        value.deserializer = class org.apache.kafka.common.serialization.ByteArrayDeserializer

2021-02-05 09:39:27,626 INFO  [org.apa.kaf.str.pro.int.ass.AssignorConfiguration] (main) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2-consumer] Cooperative rebalancing enabled now
2021-02-05 09:39:27,661 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'storage.topic' was supplied but isn't a known config.
2021-02-05 09:39:27,661 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'admin.retry.backoff.ms' was supplied but isn't a known config.
2021-02-05 09:39:27,662 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'admin.retries' was supplied but isn't a known config.
2021-02-05 09:39:27,662 WARN  [org.apa.kaf.cli.con.ConsumerConfig] (main) The configuration 'global.id.topic' was supplied but isn't a known config.
2021-02-05 09:39:27,662 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka version: 2.5.0
2021-02-05 09:39:27,662 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka commitId: 66563e712b0b9f84
2021-02-05 09:39:27,662 INFO  [org.apa.kaf.com.uti.AppInfoParser] (main) Kafka startTimeMs: 1612517967662
2021-02-05 09:39:27,747 INFO  [org.apa.kaf.str.KafkaStreams] (main) stream-client [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c] State transition from CREATED to REBALANCING
2021-02-05 09:39:27,757 INFO  [org.apa.kaf.str.pro.int.StreamThread] (databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1] Starting
2021-02-05 09:39:27,779 INFO  [org.apa.kaf.str.pro.int.StreamThread] (databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1] State transition from CREATED to STARTING
2021-02-05 09:39:27,781 INFO  [org.apa.kaf.cli.con.KafkaConsumer] (databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1) [Consumer clientId=databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-1-consumer, groupId=databus-registry] Subscribed to topic(s): global-id-topic, storage-topic
2021-02-05 09:39:27,794 INFO  [org.apa.kaf.str.pro.int.StreamThread] (databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2] Starting
2021-02-05 09:39:27,795 INFO  [org.apa.kaf.str.pro.int.StreamThread] (databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2) stream-thread [databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2] State transition from CREATED to STARTING
2021-02-05 09:39:27,814 INFO  [org.apa.kaf.cli.con.KafkaConsumer] (databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2) [Consumer clientId=databus-registry-a3977bb3-4e1b-4a4a-9617-bde4ddd55a6c-StreamThread-2-consumer, groupId=databus-registry] Subscribed to topic(s): global-id-topic, storage-topic
2021-02-05 09:39:27,794 INFO  [io.api.reg.str.StreamsRegistryConfiguration] (main) Application server gRPC: '10.1.243.45:9000'
Terminating due to java.lang.OutOfMemoryError: Java heap space
```

**Comment by jellebens** [comment]
latest-dev operator works

**Comment by cortopy** [comment]
@jellebens is latest-dev the same as `v0.0.4-v1.3.2.final` as per operatorhub.io. I'm experiencing same issue

**Comment by cortopy** [comment]
it seems that memory usage is very high on bootstrap. I'm removing the memory limits for now manually and seems to work

**Comment by jellebens** [comment]
> @jellebens is latest-dev the same as `v0.0.4-v1.3.2.final` as per operatorhub.io? I'm experiencing same issue

v 2.0 works

**Comment by jsenko** [comment]
This issue has been resolved, and will not affect the upcoming `1.0.0-v2.0.0.final` release.
